packages/satusehat/src/Adaptor.js

```diff
@@ -64,10 +68,18 @@ export function get(path, params = {}, callback = s => s) {
 
 /**
  * Make a POST request to Satusehat
- * @example
+ * @example <caption>Make a POST request to create an Organization</caption>
  * post(
  *   "Organization",
- *  { "resourceType": "Organization", "active": true,
+ *  { "resourceType": "Organization",
+ *   ...state.data
+ *  }
+ * );
+ * @example <caption>Make a POST request to create an Encounter</caption>
+ * post(
+ *   "Encounter",
+ *  { "resourceType": "Encounter",
```

Comment:
Formatting is off here too

I would actually argue that there's no point in creating an Encounter AND Organisation resources, because they're so similar. I know you've been asked for both but I don't think there's any point - not while we have nice, short examples like this